### PR TITLE
[es-ES] First batch spanish translations

### PR DESCRIPTION
**Pull request by Ekiserrepe**

### CODE REVIEW

File: @i18n/es-ES/docs/concepts/accounts/account-types.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+### Dirección de reserva comprometida
+
+Si una dirección de reserva se ve comprometida, las consecuencias son similares a las de una dirección operacional. Un actor malintencionado puede robar cualquier saldo que posea la dirección de reserva, y la institución financiera puede cambiar a una nueva dirección de reserva sin que los clientes y socios realicen ninguna acción.
+
+
+## Ver también
+
+- **Conceptos:**
+    - [Cuentas](accounts.md)
```
Comment (amarantha-k) [inline @i18n/es-ES/docs/concepts/accounts/account-types.md]:
```suggestion
    - [Cuentas](index.md)
```

File: @i18n/es-ES/docs/concepts/accounts/cryptographic-keys.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+### Código de ejemplo
+
+Los procesos de derivación de claves descritos aquí están implementados en múltiples lugares y lenguajes de programación:
+
+- En C++ en el código base de `rippled`:
+    - [Definición de semilla](https://github.com/XRPLF/rippled/blob/develop/src/ripple/protocol/Seed.h)
+    - [Derivación de clave general & Ed25519](https://github.com/XRPLF/rippled/blob/develop/src/ripple/protocol/impl/SecretKey.cpp)
+    - [Derivación de clave secp256k1](https://github.com/XRPLF/rippled/blob/develop/src/ripple/protocol/impl/SecretKey.cpp)
+- En Python 3 en [esta sección de códigos de ejemplo del repositorio]({{target.github_forkurl}}/blob/{{target.github_branch}}/content/_code-samples/key-derivation/py/key_derivation.py).
```
Comment (amarantha-k) [inline @i18n/es-ES/docs/concepts/accounts/cryptographic-keys.md]:
```suggestion
- En Python 3 en {% repo-link path="_code-samples/key-derivation/py/key_derivation.py" %}esta sección de códigos de ejemplo del repositorio{% /repo-link %}.
```

File: @i18n/es-ES/docs/concepts/accounts/addresses.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Las direcciones XRP Ledger están codificadas utilizando [base58][] con el _diccionario_ `rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz`. Como el XRP Ledger codifica varios tipos de claves con base58, antepone los datos codificados con un-byte de "prefijo de tipo" (también conocido como "prefijo de versión") para distinguirlos. El prefijo de tipo hace que las direcciones normalmente comiencen con diferentes letras en formato base58.
+
+El siguiente diagrama muestra la relación entre las claves y las direcciones:
+
+[{% inline-svg file="/docs/img/address-encoding.svg" /%}](/docs/img/address-encoding.svg "Clave pública maestra + Prefijo Tipo →  ID de cuenta + Checksum → Dirección")
+
+La fórmula para calcular direcciones XRP Ledger desde una clave pública es la siguiente. Para ver el código de ejemplo completo, consulta [`encode_address.js`](https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/address_encoding/js/encode_address.js). Para el proceso de derivar la clave pública desde una passphrase a un valor semilla, consulta [Derivación de clave](cryptographic-keys.html#key-derivation).
```
Comment (amarantha-k) [inline @i18n/es-ES/docs/concepts/accounts/addresses.md]:
```suggestion
La fórmula para calcular direcciones XRP Ledger desde una clave pública es la siguiente. Para ver el código de ejemplo completo, consulta [`encode_address.js`](https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/address_encoding/js/encode_address.js). Para el proceso de derivar la clave pública desde una passphrase a un valor semilla, consulta [Derivación de clave](cryptographic-keys.md#key-derivation).
```

File: @i18n/es-ES/docs/concepts/accounts/deleting-accounts.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Coste de eliminación
+
+**Atención:** El coste de transacción de la [transacción AccountDelete][] siempre aplica cuando la transacción está incluida en un ledger validado, incluso si la transacción falla porque la cuenta no reune los requisitos para ser eliminada. Para reducir las posibilidades de pagar un coste de transacción alto si la cuenta no puede ser eliminada, utiliza la opción `fail_hard` cuando envíes una transacción AccountDelete.
+
+A diferencia de Bitcoin y muchas otras criptomonedas, cada nueva versión de la cadena del ledger público de XRP Ledger contiene el estado completo del ledger, lo cual incrementa en tamaño con cada cuenta nueva. Por esa razón, no deberías crear nuevas cuentas XRP Ledger si no tienes necesidad. Puedes recuperar parte de los 10 XRP de la cuenta [reserva](reserves.md) eliminado la cuenta, pero destruirás por lo menos 2 XRP haciéndolo.
+
+Instituciones que reciben y envían valor en nombre de muchos usuarios pueden utilizar [**Source Tags** y **Destination Tags**](source-and-destination-tags.md) para distinguir pagos desde y para sus clientes usando una (o un puñado) de cuentas en el XRP Ledger.
```
Comment (amarantha-k) [inline @i18n/es-ES/docs/concepts/accounts/deleting-accounts.md]:
```suggestion
Instituciones que reciben y envían valor en nombre de muchos usuarios pueden utilizar [**Source Tags** y **Destination Tags**](../transactions/source-and-destination-tags.md) para distinguir pagos desde y para sus clientes usando una (o un puñado) de cuentas en el XRP Ledger.
```

**Review by amarantha-k — CHANGES_REQUESTED** [review]
Thank you for your contribution - this looks great for the first batch! 
Added suggestions to fix broken links. 
Also, recommend adding a `translations.yaml` file under the @i18n\es-ES\ similar to `https://github.com/XRPLF/xrpl-dev-portal/blob/master/%40i18n/ja/translations.yaml` before merging the first batch for Spanish translations.

**Comment by Ekiserrepe** [comment]
> Thank you for your contribution - this looks great for the first batch! Added suggestions to fix broken links. Also, recommend adding a `translations.yaml` file under the @i18n\es-ES\ similar to `https://github.com/XRPLF/xrpl-dev-portal/blob/master/%40i18n/ja/translations.yaml` before merging the first batch for Spanish translations.

Thanks for the review, I'll prepare the translations.yaml. Should I close the PR until then?

**Comment by amarantha-k** [comment]
Thank you! 
You could leave the PR open if you anticipate this work to take a few days to about a week. If you expect it to take longer, then it may be better to close the PR for now.

**Comment by Ekiserrepe** [comment]
@amarantha-k i think i added the translations.yaml file. Could you check it, please? Thank you!

**Comment by Ekiserrepe** [comment]
Added @i18n/es-ES/docs/concepts/ledgers folder and files translated.

**Comment by amarantha-k** [comment]
@Ekiserrepe Thank you! I was able to test locally with the `translations.yaml` file you added.

**Comment by amarantha-k** [comment]
FYI, waiting on a few localization fixes from Redocly - #2524, #2516, #2425

**Comment by Ekiserrepe** [comment]
Added /networks-and-servers translations

File: redocly.yaml [hunk 5]
```diff
@@ -11,6 +11,8 @@ i18n:
       name: English
     - code: ja
       name: 日本語
+    - code: es-ES
```
Comment (amarantha-k) [inline redocly.yaml]:
```suggestion
```

File: redocly.yaml [hunk 6]
```diff
@@ -11,6 +11,8 @@ i18n:
...
+      name: Español
```
Comment (amarantha-k) [inline redocly.yaml]:
```suggestion
```

**Review by amarantha-k — CHANGES_REQUESTED** [review]
Added suggestions to remove the locale information from `redocly.yaml` for now. That way, we can merge the PR to add the translated pages to the master branch. 
As you continue to translate more documentation pages, you can create new PRs that can be reviewed and merged sooner. Once we have a majority of the content translated, we can enable the language dropdown for Spanish on the site by re-adding the locale in the `redocly.yaml` file.

**Review by amarantha-k — APPROVED** [review]
Thanks for incorporating the suggestions. Local build was successful.

**Pull request merged**